source/draw/gpu/opengl/OpenGLCanvas.ooc

```diff
@@ -28,6 +28,89 @@ OpenGLCanvas: class extends OpenGLSurface {
 	_target: OpenGLPacked
 	_renderTarget: GLFramebufferObject
 	context ::= this _context as OpenGLContext
+	// TODO: Remove once rendering is done in normalized coordinates
```

Comment:
Converting to reference again in just to decode back to normalized in the vertex shader is messy but first it should start working on top of existing solutions since transform is harder to test.